keras_nlp/metrics/edit_distance_test.py

```diff
@@ -107,34 +107,30 @@ def test_rank_2_tensor_input_normalize_false(self):
 
     def test_model_compile_normalize(self):
         inputs = keras.Input(shape=(None,), dtype="string")
-        outputs = tf.strings.lower(inputs)
```

Comment:
Isn't it OK to keep TF code if we assert TF backend?

Reply:
It is not. In keras-core, a symbolic input like `keras.Input` cannot be called on tf ops. You would need to wrap this with a layer. Also, none of the tests really seemed to be doing anything with the lower, it was just a holdover.

Reply:
Here's [the error](https://github.com/keras-team/keras-core/blob/ce9b511a063f0be3988ba34accc2e07dc1812ab9/keras_core/backend/common/keras_tensor.py#L87-L108) you would get in `keras_core` tf backend btw.

This will be an interesting "gotcha" to see how many people hit when moving to keras-core, but I think that's a question that is out of scope for this PR.